src/test/java/teammates/test/cases/storage/FeedbackResponsesDbTest.java

```diff
@@ -103,13 +103,13 @@ public void testTimestamp()
 
         String newRecipientEmailTwo = "new-email-two@tmt.com";
         feedbackResponse.recipient = newRecipientEmailTwo;
-        frDb.updateFeedbackResponse(feedbackResponse, true);
+        frDb.updateFeedbackResponse(feedbackResponse);
 
         FeedbackResponseAttributes updatedFrTwo =
                 frDb.getFeedbackResponse(feedbackQuestionId, giverEmail, newRecipientEmailTwo);
 
-        // Assert lastUpdate has NOT changed.
```

Comment:
You should remove the test rather than change the test.

Reply:
Done!